optuna/dashboard.py

```diff
@@ -57,7 +58,7 @@
 </p>
 """
 
-_DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+_DATETIME_FORMAT: str = "%Y-%m-%d %H:%M:%S"
```

Comment:
I'm wondering if we should avoid introducing type hints to variables that didn't used to have comment style type hints. We usually only add type hints where mypy would complain assuming that it's clever to detect possible bug-prone code (it get a bit verbose and may hurt readability too adding type hints). It also seems a bit orthogonal to the motivation behind #1897 but sorry, this was probably not communicated well enough. What do you think?

Reply:
(Although I have added a lot of annotations already,) I have to admit that I strongly agree with you that too many annotations lower readability, and it sounds reasonable to add variable annotations only to variables which already have comment style annotations. I'll roll back some commits if you're ok with it!

Reply:
If we add anntations only to variables which already have comment style annotations, we can just use `com2ann` suggested by @harupy in the issue all at once!

Reply:
Sure, that might be better using `com2ann`. Do you want to close this PR and create a new one?

Reply:
Sure.

Reply:
Looking forward, thanks!